R/countdown.R

```diff
@@ -300,33 +306,72 @@ countdown_fullscreen <- function(
   )
 }
 
+# ---- Style Countdown ----
+
+#' @describeIn countdown Set global default countdown timer styles using CSS.
+#'   Use this function to globally style all countdown timers in a document or
+#'   app. Individual timers can still be customized.
+#' @param .selector In `style_countdown()`: the CSS selector to which the styles
+#'   should be applied. The default is `:root` for global styles, but you can
+#'   also provide a custom class name to create styles for a particular class.
+#' @export
+style_countdown <- function(
```

Comment:
Thanks for calling this out. `style_{thing}` is a pattern I tend to following across my projects, even though it breaks the usual prefixing rules. In Shiny/html land, I tend to prefer a verb-first approach to separate between functions that _create_ UI and functions for interactivity.

For example, looking at this list I'd prefer `update_countdown()` and `style_countdown()`, since in Shiny we group `update_` functions and in my packages I use `style_` for prefixes.

The problem -- and the reason I didn't use `update_countdown()` when I wrote it -- is `countdown_action()`. I can't think of a good name for that function. The other best option I've come up with is `trigger_countdown()`, which I don't like enough to use. So, IIRC, I went with `countdown_update()` for internal consistency.

So, on the whole, I agree and think `countdown_style()` is probably the best option.

Reply:
@gadenbuie We can also use this as an opportunity to rename functions if you wish. Redirects can be setup for backward compatibility with `lifecycle`.